packages/grid/x-data-grid/src/components/menu/columnMenu/default/GridColumnMenuDefaultContainer.tsx

```diff
@@ -0,0 +1,61 @@
+import clsx from 'clsx';
```

Comment:
Yes you are right, I also thought to keep one container, but since we still need to work on accessibility and keyboard navigation and some logic could be different for both the menus, but yeah, it makes sense too, I'll make them one for now 👍